### DISCUSSION TITLE
Multiple select within custom field

### DISCUSSION THREAD

**Discussion by carlbannister** [liedekef/events-made-easy#297] [Q&A] [answered]
Hi I currently have a custom field to declare the status of the account held with our governing body, it is used in people view. This is a dropdown field that has 3 options , Active , None , Expired. All is working as it should but when in the people view and i sort by this field i would like to multiple select say Active then delete for all people meeting this criteria. Is this possible?

**Comment by liedekef** [comment]
In the people overview, you can filter on a custom field and value:
![image](https://github.com/liedekef/events-made-easy/assets/75667/2dd301dc-3ae5-4b14-8edc-67bc23b0fa96)

When defining the field, make sure to set "Searchable or sortable" to "yes" and then you can select that field in the "custom fields to filter on" box. In the "Custom field value" you then enter the value of choice and presto :-)

**Reply by carlbannister** [reply]
OK, but can i change multiple entries in one go from say Active to expired? and not have to go through each person individually

**Reply by liedekef** [reply, marked as answer]
Sorry, but currently not possible ... that would require extra dev

**Reply by carlbannister** [reply]
No worries thanks
